native/objcexport-header-generator/src/org/jetbrains/kotlin/backend/konan/objcexport/stubs.kt

```diff
@@ -45,6 +46,17 @@ val ObjCExportStub.psiOrNull
 
 abstract class ObjCTopLevel : ObjCExportStub
 
+class ObjCNSEnum(
```

Comment:
It would be reasonable to make it also have an explicit Swift name using the `swift_name` attribute, just like it is done for the rest of the declaration classes.